### ISSUE TITLE
Feature : Initially Shows the current location weather.............

### ISSUE THREAD

**Issue by Sakshamjain98** [1010varun/weather-app#41]

**Comment by Sakshamjain98** [comment]
Sir please assign me this issue under IWOC

**Comment by 1010varun** [comment]
So by adding this, you have to remove the button that fetches weather based on the current location. Also how will you manage if user will not provide the access to current location??

**Comment by Sakshamjain98** [comment]
Yeah u are right so we have to first take the permission white opening the web app the shows the weather if yes and if no by the user we will provide as same this time now

**Comment by 1010varun** [comment]
assigning it to you
